src/midi.js

```diff
@@ -36,7 +47,17 @@ const midi = (notes, fileName) => {
 		}
 	});
 
-	fs.writeFileSync(fileName, file.toBytes(), 'binary');
+	let outputFile = path.join(process.cwd(), '/Output/', fileName + '.mid');
```

Comment:
1. Since we already have mkdirp, we can avoid hardcoding a output folder name. We can set it to `/` instead and let the user decide if they want a output folder or not. Also they can set whatever name they want (mkdirp should take care of this). 
2. I like the suggestion here for not having to specify the mid extension, but we must add support for folks who prefer to add the .mid extension. Basically we should append .mid only if it s not already there.